app-editors/scite/scite-4.1.2.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs flag-o-matic gnome2-utils xdg-utils
```

Comment:
You'll have to fix this commit, so you can sort this list alphabetically to improve ebuilds readability.